doc/api/documentation.md

```diff
@@ -32,6 +32,17 @@ The stability indices are as follows:
 > [semantic versioning][] rules. Non-backward compatible changes or removal may
 > occur in any future release. Use of the feature is not recommended in
 > production environments.
+>
+> Experimental features are subdivided into stages:
+>
+> * 1.0 - Early development. Experimental features at this stage are unfinished
+>   and subject to substantial change. They are not yet ready for user testing.
+> * 1.1 - Active development. Experimental features at this stage are nearing
+>   potential stability, with only minor changes expected.
+> * 1.2 - Release candidate. Experimental features at this stage are hopefully
+>   ready to become stable. No further breaking changes are anticipated and we
+>   welcome user testing and feedback so that we can know that this feature is
```

Comment:
Is there a point in the development where we do not welcome user testing and feedback?

Reply:
> Is there a point in the development where we do not welcome user testing and feedback?

Yes. What the discussion was calling “alpha” are features that are unfinished, and so it’s counterproductive to receive bug reports for those. Users can always provide _feedback,_ like to suggest API changes or something, but I don’t think we need to mention that (or we can, if you want to distinguish).

Reply:
I don't think it's fair to call any collaboration made in good faith "counterproductive", and in my experience the problem is way more often a lack of reviews/feedback than a tsunami of issues. I personally welcome feedback and user testing of everything I do on Node.js, whatever the stage, and I'm -1 having docs that say otherwise.

Reply:
I’m fine with removing the line about no feedback for early features. We can always add it back if we need to.

Reply:
I revised per the discussion here.